### PR TITLE
Update troubleshooting issues for #2613

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <MICGIBSO@uk.ibm.com>

PR for 0.11.0 troubleshooting updates on https://github.com/eclipse/codewind/issues/2613

### CODE REVIEW

File: docs/_documentations/troubleshooting.md [hunk 1]
```diff
@@ -384,13 +384,13 @@ Sometimes when a new project is created, it doesn't show up in the hierarchy vie
...
-Issue link:
-18.10:
+Issue link: Updated in https://github.com/eclipse/codewind/issues/2613
+0.11.0: Updated for this release.
 -->
 ## Context Root / Application Endpoint not correct
 If you create or bind a project that has a context root set in `.cw-settings`, such as a project using the Lagom template, the context root is not picked up initially. This also happens after restarting Codewind.
 
-**Workaround** For Eclipse, add the context root to the URL in your browser. For example, the browser might open with `localhost:34567` instead of `localhost:34567/mycontextroot`, so type `mycontextroot`. For VS Code and Che, edit and save the `.cw-settings` file, and the context root updates.
+**Workaround** For a permanent fix for all IDEs, edit and save the `.cw-settings` file, and the context root updates. For a temporary workaround for Eclipse, add the context root to the URL in your browser. For example, the browser might open with `localhost:34567` instead of `localhost:34567/mycontextroot`, so type `mycontextroot`.
```
Comment (tetchel) [inline docs/_documentations/troubleshooting.md:393]: The temporary workaround applies to all IDEs too.

File: docs/_documentations/troubleshooting.md [hunk 2]
```diff
@@ -206,7 +206,7 @@ Issue link: https://github.com/eclipse/codewind/issues/243
 ## Adding an existing Open Liberty project fails to build because of missing files
 An Open Liberty project fails to build after it is added into Codewind with the **Add Existing Project** action, and the project fails to build because of missing files.
 
-**Workaround:** Bind the existing project again and click **No** followed by **Other** for the project type.
+**Workaround:** Bind the existing project again but this time, don't accept the detected type. Click **No** for VSCode, **Next** for Eclipse, and then choose **Other (Codewind Basic Container)** for the project type.
```
Comment (tetchel) [inline docs/_documentations/troubleshooting.md:209]: IntelliJ should have a flow like this too, but I'm not sure what the button would say. We could simplify this by not specifying exactly which buttons have to be pressed in each IDE.

File: docs/_documentations/troubleshooting.md (hunk 1 above)
Comment (micgibso) [reply docs/_documentations/troubleshooting.md:393]: Adjusted to `For a temporary workaround, ...`

File: docs/_documentations/troubleshooting.md (hunk 2 above)
Comment (micgibso) [reply docs/_documentations/troubleshooting.md:209]: Reworded

**Pull request merged**